source/Core/UtilsUrl.php

```diff
@@ -303,14 +304,7 @@ public function getActiveShopHost()
      */
     public function extractHost($url)
     {
-        $host = $url;
-        $sUrlHost = $this->parseUrlAndAppendSchema($host, PHP_URL_HOST);
-
-        if (!is_null($sUrlHost)) {
-            $host = $sUrlHost;
-        }
-
-        return $host;
+        return $this->parseUrlAndAppendSchema($url, PHP_URL_HOST) ?: $url;
```

Comment:
Shouldn't condition be: `!is_null($this->parseUrlAndAppendSchema($url, PHP_URL_HOST)) ?: $url;` ?